packages/bot-web-ui/src/components/bot-footer-extensions/security-and-privacy.tsx

```diff
@@ -3,7 +3,7 @@ import { Icon, Popover, StaticUrl } from '@deriv/components';
 import { localize } from '@deriv/translations';
 
 const SecurityAndPrivacy: React.FC = () => (
-    <Popover alignment='top' message={localize('Security and privacy')}>
+    <Popover alignment='top' zIndex='9999' message={localize('Security and privacy')}>
```

Comment:
```suggestion
    <Popover alignment='top' zIndex='9999' message={localize('Security and privacy')}>
```
Pretty high `z-index` 👀

Reply:
done ✅ kept z-index higher then the component i need it to come